docs/domains/Domain.md

```diff
@@ -23,6 +23,7 @@ DomainSpec is a description of a domain.
 | `domainHome` | string | The folder for the WebLogic Domain. Not required. Defaults to /shared/domains/domains/domainUID if domainHomeInImage is false Defaults to /u01/oracle/user_projects/domains/ if domainHomeInImage is true |
 | `domainHomeInImage` | Boolean | True if this domain's home is defined in the docker image for the domain. Defaults to true. |
 | `domainUID` | string | Domain unique identifier. Must be unique across the Kubernetes cluster. Not required. Defaults to the value of metadata.name |
+| `failBootOnSituationalError` | Boolean | If true (the default), on WebLogic server that supports this feature, the WebLogic server  boot would fail if any errors occur when applying situational configuration  during server startup. If false, WebLogic server would start if there are errors in the situational configuration files, but some configuration overrides may be skipped. |
```

Comment:
(suggested re-write) In WebLogic Server versions that support this feature (greater than 12.2.1.3.0):
<ul><li>If true (the default), then WebLogic Server  would fail to boot if any errors occurred when applying the situational configuration  during server startup. </li><li>If false, and if there were errors in the situational configuration files, then WebLogic Server would start but some configuration overrides may be skipped.</li></ul>

Reply:
The `failBootOnSituationalError` attribute has been removed in the latest update